src/test/java/org/fundacionjala/at15/pokemon/io/QueryTest.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.at15.pokemon.io;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import com.google.gson.Gson;
+
+import static org.fundacionjala.at15.pokemon.io.PathHandler.*;
+import static org.fundacionjala.at15.pokemon.io.Query.*;
+import org.fundacionjala.at15.pokemon.*;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+public class QueryTest {
+    @Test
+    public void getFilesTest() throws IOException {
```

Comment:
This Unit Test is throwing an Exception when I run the tests with gradlew.
I think, you should implement a 'try-catch' function instead 'throws IOException'